tests/Builders/InfoBuilderTest.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Vyuldashev\LaravelOpenApi\Tests\Builders;
+
+use Vyuldashev\LaravelOpenApi\Builders\InfoBuilder;
+use Vyuldashev\LaravelOpenApi\Tests\TestCase;
+
+class InfoBuilderTest extends TestCase
+{
+    /**
+     * @dataProvider providerBuildContact
+     * @param array $config
+     * @param array $expected
+     * @return void
+     */
+    public function testBuildContact(array $config, array $expected): void
+    {
+        $SUT = new InfoBuilder();
```

Comment:
What does `$SUT` mean?

Reply:
`SUT` is System Under Test.
In this case, it's `InfoBuilder` class.